tests/test_parsers.py

```diff
@@ -41,7 +30,7 @@ def test_gelfparser_parse_raw_file(gelf_logger):
     gelf_logger.info('{"username": "foo"}')
     gelf_logger.info('{"username": ""}')
 
-    events = list(parser.parse(gelf_logger.handlers[0].stream.name))
+    events = list(parser.parse(open(gelf_logger.handlers[0].stream.name)))
```

Comment:
Always use the `with` statement to ensure the file is closed once parsed:

```suggestion
    with open(gelf_logger.handlers[0].stream.name) as stream:
        events = list(parser.parse(stream))
```